docs/reference/cluster_manifest.md

```diff
@@ -254,6 +254,8 @@ explanation of `ttl` and `loop_wait` parameters.
 * **synchronous_mode_strict**
   Patroni `synchronous_mode_strict` parameter value. Can be used in addition to `synchronous_mode`. The default is set to `false`. Optional.
 
+* **synchronous_node_count**
```

Comment:
Should add the info here that it's only available for Spilo images with Patroni 2.0+

Reply:
It should be right here at 257 or in description in 258?

Reply:
Should I add ❗ before notification?